python/lsst/ip/isr/isrTask.py

```diff
@@ -2091,10 +2105,11 @@ def maskDefect(self, exposure, defectBaseList):
             defectList = defectBaseList
         defectList.maskPixels(maskedImage, maskName="BAD")
 
-        if self.config.numEdgeSuspect > 0:
+        numEdgeSuspect = numpy.max([numEdgeSuspect, self.config.numEdgeSuspect])
```

Comment:
Do we want to do this calculation here as well as at line 1301, above?

At line 1301, it seems appropriate. Here it's saying that even if I call this particular method with a very specific value of `numEdgeSuspect` for my own reasons, the code might just ignore it and use the task config instead. That seems like it could be unfortunate.

I'd suggest that if you want to leave this behaviour in, you might want to change the docstring above to indicate that `numEdgeSuspect` can be ignored in some circumstances.

Reply:
Moving to a stand-alone method removes this multiple calculation.